types/modules.d.ts

```diff
@@ -445,6 +445,9 @@ declare module "services/floating-manager/floating-manager" {
 declare module "services/toast-manager/models/toast-severity" {
     export type ToastSeverity = 'Info' | 'Success' | 'Warning' | 'Error';
 }
+declare module "services/toast-manager/models/toast-type" {
```

Comment:
did you update this file manually or did you run types:generate ?

Reply:
manually